programs/survival/README.md

```diff
@@ -296,6 +296,7 @@ Various scripts that modify various game elements, often replicating popular mod
     It will display a number in the scoreboard. Divide said number by 100 to get actual speed.
     Speed toggle in minihud is obviously much better.
 
+
```

Comment:
Missing input in the readme?

Reply:
I moved the app from `survival/` into `utilities/`, the app feels more like a utility then a survival feature. so I decided to remove the survival readme entire I created. Apparently I left a new line. 
I was thought about adding it to the utilities readme entire, however that file is currently empty. so I was uncertain how welcome setting that precedent would be. In hindsight i should have asked about it on the discord server.

Would it be cool if I added the readme entire into the utilities readme file and formatted it like the survival one?

Reply:
There is a readme being added to utilities in #289, please don't add one here or they will conflict.

If you want though you can leave the description here and it can be added later.

Reply:
I assume 'here' means here in the `survival/README.md`

Reply:
I mean in this PR, that PR linked adds a readme to utilities.

Reply:
I added a description to the survival readme as well as in the head of the spellbook script